events/management/commands/update_n_events.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management import BaseCommand, CommandError
+from django.db import transaction
+
+from events.models import Keyword, Place
+from events.utils import recache_n_events_in_locations, recache_n_events
+
+
+class Command(BaseCommand):
+    help = "Update keyword and place event numbers"
+    models = {'keyword': Keyword,
+              'place': Place}
+
+    def add_arguments(self, parser):
+        parser.add_argument('model', nargs='?', default=False)
+        parser.add_argument('--all',
+                            default=False,
+                            action='store_true',
+                            dest='update_all',
+                            help='Recalculate everything from scratch')
+
+    def handle_keywords(self, update_all=False):
+        if update_all:
+            keywords = Keyword.objects.all()
+        else:
+            keywords = Keyword.objects.filter(n_events_changed=True)
+        recache_n_events(keywords)
+        print("Updated %s keyword event numbers." % ('all' if update_all else 'changed'))
+        print("A total of %s keywords updated." % (str(keywords.count())))
+
+    def handle_places(self, update_all=False):
+        if update_all:
+            places = Place.objects.all()
+        else:
+            places = Place.objects.filter(n_events_changed=True)
+        recache_n_events_in_locations(places)
+        print("Updated %s place event numbers." % ('all' if update_all else 'changed'))
+        print("A total of %s places updated." % (str(places.count())))
+
+    def handle(self, *args, model=None, update_all=False, **kwargs):
+        if model and model not in self.models:
```

Comment:
`if model and model not in ('keyword', 'place')` if you get rid of `self.models`.